HW1/P6/P6.txt

```diff
@@ -1 +1,5 @@
 # Your discussion here
+
+This was by far the best problem of the homework set! I had so much fun running the shakespeare sentance generated over and over. Here is my favorite output, it pretty much sums up this homework:
+
+Direct. All these the wretches that we find, we stoop and humble
```

Comment:
Yep, this was a fun problem. I think we had to include 10 phrases with 20 words each in this file though.